src/internal/operations/backup.go

```diff
@@ -239,35 +256,24 @@ func (op *BackupOperation) Run(ctx context.Context) (err error) {
 		return op.Errors.Failure()
 	}
 
-	// force exit without backup in certain cases.
-	// see: https://github.com/alcionai/corso/pull/2510#discussion_r1113532530
-	for _, e := range op.Errors.Recovered() {
```

Comment:
Note: I've removed this section. Reasons:
1. There is 1:1 mapping between `AddRecoverable` and `LabelForceNoBackupCreation`, i.e. we always attach this label to recoverable failures. Now that we *want* to persist backups which have recoverable failures, we need to reassess places where this label should be added. 
2. We don't hit this code path currently. Assuming `FailAfterRecovery` policy is in place(which is default), `finalizeErrorHandling` will [fold recoverable failures](https://github.com/alcionai/corso/blob/e0368ba45af72be2d97e7e82285f76bba84c0636/src/internal/operations/helpers.go#L40) into non-recoverable. Later on, `persistResults` is going to return this folded `Failure()` and we will exit at line 239.